packages/rancher-vsphere-csi/charts/values.yaml

```diff
@@ -70,6 +81,7 @@ csiNode:
     livenessProbe:
       repository: rancher/mirrored-k8scsi-livenessprobe
       tag: v2.2.0
+  tolerations: {}
```

Comment:
why did we introduce tolerations here?

Reply:
I left that in there by mistake, I'll remove it.